packages/local-environment/.env.example

```diff
@@ -5,8 +5,18 @@ PRIVATE_KEY_3=3000000000000000000000000000000000000000000000000000000000000003
 PRIVATE_KEY_4=4000000000000000000000000000000000000000000000000000000000000004
 PRIVATE_KEY_5=5000000000000000000000000000000000000000000000000000000000000005
 PRIVATE_KEY_6=6000000000000000000000000000000000000000000000000000000000000006
-PRIVATE_KEY_7=6000000000000000000000000000000000000000000000000000000000000006
-PRIVATE_KEY_8=6000000000000000000000000000000000000000000000000000000000000006
-PRIVATE_KEY_9=6000000000000000000000000000000000000000000000000000000000000006
+PRIVATE_KEY_7=7000000000000000000000000000000000000000000000000000000000000007
+PRIVATE_KEY_8=8000000000000000000000000000000000000000000000000000000000000008
+PRIVATE_KEY_9=9000000000000000000000000000000000000000000000000000000000000009
 
-AGENTS_IMAGE=gcr.io/nomad-xyz/nomad-agent:prestwich-remove-deploy-gas
\ No newline at end of file
+AGENTS_IMAGE=gcr.io/nomad-xyz/nomad-agent:prestwich-remove-deploy-gas
+
+WETH_ADDRESS=
+
+# Only set the following if you're forking mainnet
+
+ALCHEMY_API_KEY=UEz-5kiUuyGqZhi8SNUrgJrEy-LPqkZK
+BLOCK_NUMBER=15670813
+
+tDomainNumber=1
+jDomainNumber=2
```

Comment:
Why do we need to have those in `.env`? or generally set up outside of the test case scenario?

Reply:
In case of network forking, users will then be able to put in a domain number that matches with the real chain ID (but it's honestly not that important and can be left out - lmk if you think this is the better option)

Reply:
lets get rid of it please. It just create additional fuzz. If they want they can set it in particular test, especially that they will define forking in that test.

Reply:
Removed